docs/recipes/Code-Splitting-Using-Holocron.md

```diff
@@ -189,40 +189,35 @@ root
 
 > To view a module chunks example using dynamic importing, [check out the sample module "Franks Burgers"][franks-burgers]
 
-### Using `holocronModule`
+### Using `Module.holocron`
 
 `src/Module.jsx`
 ```jsx
 import React from 'react';
-import { holocronModule } from 'holocron';
 
 let Chunk = () => <p>Loading...</p>;
 
 export function MyHolocronModule() {
   return <Chunk />;
 }
 
-export default holocronModule({
+MyHolocronModule.holocron = {
   name: 'holocron-module-name',
-  // the load property allows us to load our chunk with our module
-  load: () => () => Promise.all([
-    // the same dynamic import is used to create a promise with our chunk
-    import(/* webpackChunkName: '<chunkName>' */ './Chunk')
-      // we check if our chunk has a `default` export or not
-      .then((imported) => imported.default || imported)
-      // and assign the exported chunk to memory
-      .then((Component) => {
-        Chunk = Component;
-      })
-      // in the event of failure
-      .catch((error) => {
-        Chunk = () => <p>{error.message}</p>;
-      }),
-  ]),
+  // the loadModuleData property allows us to load our chunk with our module
+  loadModuleData: async () => {
+    try {
+      // the same dynamic import is used to create a promise with our chunk
+      const imported = await import(/* webpackChunkName: '<chunkName>' */ './Chunk');
+      Chunk = imported.default || imported;
+    } catch (error) {
+      Chunk = () => <p>{error.message}</p>;
+    }
+  },
   // we can wait for our holocron module to load the chunk during server-side rendering
   // and render the module with the loaded chunk
-  options: { ssr: true },
```

Comment:
this was only relevant to the `load` function right ? is there a case to disable calling `loadModuleData` on the server ?

Reply:
Yes those were only for `load`. 

And Yes:
```js
MyModule.holocron = {
  loadModuleData: global.BROWSER ? loadModuleData : undefined;
}
```